src/images/opengl/castleglshaders.pas

```diff
@@ -1778,6 +1783,34 @@ procedure TGLSLProgram.AttachGeometryShader(const S: string);
   AttachShader(stGeometry, S);
 end;
 
+procedure TGLSLProgram.SetTransformFeedbackVaryings(const Varyings: array of PChar; const IsSingleBufferMode: Boolean);
+var
+  TransformFeedbackBufferMode,
+  ErrorCode: TGLuint;
+  VaryingLength: Cardinal;
+begin;
+  {$ifndef OpenGLES}
```

Comment:
Since we make an exception on desktops when transform feedback is not supported, it seems consistent to also make an exception on OpenGLES (always, when `VaryingLength > 0`) like

```pascal
 raise EGLSLTransformFeedbackError.Create('Transform feedback not supported by OpenGLES');
```